### PR TITLE
Snippet lib release v1.0.1.

### PR DESCRIPTION
**Pull request by adorokhine**

### CODE REVIEW

File: third_party/sl4a/gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This version code implements the versioning recommendations in:
 # https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
-VERSION_CODE=1000100
-VERSION_NAME=1.0.1-SNAPSHOT
+VERSION_CODE=1000199
```
Comment (xpconanfan) [inline third_party/sl4a/gradle.properties]: i'm not familiar with how this works... Where did '99' come from?...

File: third_party/sl4a/gradle.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+VERSION_NAME=1.0.1
```
Comment (xpconanfan) [inline third_party/sl4a/gradle.properties]: so, the version name was 1.0.1 already?...

File: third_party/sl4a/gradle.properties (hunk 1 above)
Comment (adorokhine) [inline third_party/sl4a/gradle.properties]: 99 is the RC name of the final release for a particular version code. You should read the blog link mentioned in the comment right above this line :)

File: third_party/sl4a/gradle.properties (hunk 2 above)
Comment (adorokhine) [inline third_party/sl4a/gradle.properties]: The version was 1.0.1-SNAPSHOT (which is maven/gradle speak for prerelease) and now 1.0.1 is released:
http://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it

File: third_party/sl4a/gradle.properties (hunk 1 above)
Comment (xpconanfan) [inline third_party/sl4a/gradle.properties]: The link gives me a 404 not found. Need to update link?

Comment (xpconanfan) [inline third_party/sl4a/gradle.properties]: Nvm, wrong link..

**Pull request merged**